plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/TestMongoClientConfig.java

```diff
@@ -123,4 +148,33 @@ public void testSpecialCharacterCredential()
         MongoCredential expected = MongoCredential.createCredential("username", "database", "P@ss:w0rd".toCharArray());
         assertEquals(credential, expected);
     }
+
+    @Test
+    public void testValidation()
+            throws Exception
+    {
+        Path keystoreFile = Files.createTempFile(null, null);
+        Path truststoreFile = Files.createTempFile(null, null);
+
+        assertFailsValidation(
+                new MongoClientConfig().setKeystorePath(keystoreFile.toFile()),
+                "validTlsConfig",
+                "'mongodb.tls.keystore-path', 'mongodb.tls.keystore-password', 'mongodb.tls.truststore-path' and 'mongodb.tls.truststore-password' must be empty when TLS is disabled",
```

Comment:
nit : Sorry I mentioned in the wrong place - we can have a extracted method which could use `assertFailsValidation`